.github/workflows/create-release.yaml

```diff
@@ -0,0 +1,91 @@
+name: Create Release
+run-name: Create Release ${{ github.ref_name }}
+
+env:
+  IMAGE_REPO: europe-docker.pkg.dev/kyma-project/prod/eventing-publisher-proxy
+
+on: workflow_dispatch
+
+jobs:
+  verify-release:
+    name: Verify image version
+    runs-on: ubuntu-latest
+    outputs:
+      release_version: ${{ steps.release-version.outputs.release_version }}
+    steps:
+      - name: Checkout EPP repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Verify that the current branch has a name that starts with 'release-'
+        run: |
+          CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+          if [[ "$CURRENT_BRANCH" == release-* ]]; then
+            echo "current_branch=$CURRENT_BRANCH" >> $GITHUB_ENV
+            echo "Branch name starts with 'release-'."
+          else
+            echo "Branch name does not start with 'release-'."
+            exit 1
+          fi
+
+      - name: Get the next release version
+        id: release-version
+        run: |
+          RELEASE_VERSION=$(./scripts/get_next_release_version.sh "$current_branch")
+          echo "release_version=$RELEASE_VERSION" >> $GITHUB_OUTPUT
+
+      - name: Check image tag
+        env:
+          RELEASE_VERSION: ${{ steps.release-version.outputs.release_version }}
+        run: ./scripts/check_tag_info.sh $RELEASE_VERSION
+
+  create-draft:
+    name: Create the draft release
+    needs: verify-release
+    runs-on: ubuntu-latest
+    env:
+      RELEASE_VERSION: ${{ needs.release-version.outputs.release_version }}
+    outputs:
+      release_id: ${{ steps.create-draft.outputs.release_id }}
+    steps:
+      - name: Checkout EPP repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Create changelog
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: ./scripts/create_changelog.sh $RELEASE_VERSION
+
+      - name: Create the draft release
+        id: create-draft
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          RELEASE_ID=$(./scripts/create_draft_release.sh $RELEASE_VERSION)
+          echo "release_id=$RELEASE_ID" >> $GITHUB_OUTPUT
+
+      - name: Add lightweight tag
```

Comment:
This step happens to trigger the prow build job and I remember that it was not so easy to see that when I looked at this the first time. Why don't we leave a nice comment here that points this out?

Reply:
Done